README.md

```diff
@@ -56,17 +59,17 @@ with a fresh install of Ubuntu 20.04.
 You can install the new CLI using the install script:
 
 ```console
-curl -L https://raw.githubusercontent.com/docker/aci-integration-beta/main/scripts/install_linux.sh | sh
+curl -L https://raw.githubusercontent.com/docker/cloud-integration-beta/main/scripts/install_linux.sh | sh
```

Comment:
We'll need to update the script as well if you want to rename the binaries lower down